make/autoconf/basic.m4

```diff
@@ -390,6 +394,9 @@ AC_DEFUN_ONCE([BASIC_SETUP_OUTPUT_DIR],
   AC_SUBST(OUTPUTDIR)
   AC_SUBST(CONFIGURESUPPORT_OUTPUTDIR)
 
+  AC_SUBST(CC_NOFIXPATH)
+  AC_SUBST(CXX_NOFIXPATH)
```

Comment:
These should appear in a 'closed' file.

Reply:
This is to support `spec.gmk` generation.
```
  SPEC="$OUTPUTDIR/spec.gmk"
...
  AC_SUBST(CC_NOFIXPATH)
  AC_SUBST(CXX_NOFIXPATH)

  # The spec.gmk file contains all variables for the make system.
  AC_CONFIG_FILES([$OUTPUTDIR/spec.gmk:$AUTOCONF_DIR/spec.gmk.in])
```
It could be included in a `closed` file but that file need to be included here hence it doesn't seem much different.